src/neuroconv/datainterfaces/ecephys/requirements.txt

```
@@ -1,2 +1,2 @@
-spikeinterface>=0.98.2
+spikeinterface[qualitymetrics]>=0.99.1
```

```suggestion
spikeinterface>=0.99.1
```

And then reducing this back to minimal for basic default usage (then the special quality metric is available for people using SI that know what they're doing, but is optionally imported and used by our tools)

---

Done